spec/models/volunteer_op_form_spec.rb

```diff
@@ -6,23 +6,21 @@
   describe '#save' do
     context 'with post to doit flag' do
       it 'publish volunteer op to doit' do
+        ActiveJob::Base.queue_adapter = :test
         org = create(:organisation)
 
-        input_params = {"title"=>"test007-001",
-                         "description"=>"description for 007",
-                         "address"=>"1234",
-                         "postcode"=>"HA1 3UJ",
-                         "post_to_doit"=>"1",
-                         "advertise_start_date"=>"",
-                         "advertise_end_date"=>"",
-                         "organisation_id" => org.id}
+        input_params = {'title'=>'test007-001',
+                         'description'=>'description for 007',
+                         'address'=>'1234',
+                         'postcode'=>'HA1 3UJ',
+                         'post_to_doit'=>'1',
+                         'advertise_start_date'=>'',
+                         'advertise_end_date'=>'',
+                         'organisation_id' => org.id}
         doit_volunteer_op = VolunteerOpForm.new(input_params)
         allow(doit_volunteer_op).to receive(:valid?).and_return(true)
-        allow(PostToDoitJob).to receive(:perform_later)
-
         doit_volunteer_op.save
-
-        expect(PostToDoitJob).to have_received(:perform_later)
+        expect {PostToDoitJob.perform_later}.to have_enqueued_job(PostToDoitJob)
       end
```

Comment:
@tansaku please take a look to this changes carefully to see if I'm testing to what it was being tested originally, meaning if I interpreted correctly what we want to test here.

Reply:
it definitely looks fine, although we can never be sure.  I think we'd have to manually test to be certain, but this functionality is not under heavy use, so I don't think we can justify any more time looking at this at the moment ...